examples/tree_1d_dgsem/elixir_shallowwater_source_terms.jl

```diff
@@ -0,0 +1,60 @@
+
+using OrdinaryDiffEq
+using Trixi
+using TrixiShallowWater
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterEquationsWetDry1D(gravity_constant = 9.81)
+
+initial_condition = initial_condition_convergence_test
+
+###############################################################################
+# Get the DG approximation space
+
+volume_flux = (flux_wintermeyer_etal, flux_nonconservative_wintermeyer_etal)
+solver = DGSEM(polydeg = 3,
+               surface_flux = (flux_lax_friedrichs, flux_nonconservative_fjordholm_etal),
```

Comment:
Yes, but only if we also use `(flux_wintermeyer_etal, flux_nonconservative_ersing_etal)` at the surface. 
Currently `flux_nonconservative_ersing_etal` should not compatible with `flux_nonconservative_fjordholm_etal` at the surface. This is because the other nonconservative fluxes are not consistent and produce an extra volume contribution, that is then canceled with the additional surface term `equations.gravity * h_ll * b_ll`. They should work together if remove the extra inner contribution from `flux_nonconservative_fjordholm_etal`.